dagfactory/dagbuilder.py

```diff
@@ -331,6 +331,18 @@ def set_dependencies(
                     ] = tasks_and_task_groups_instances[dep]
                     source.set_upstream(dep)
 
+    def build_fallback_dag(self, error: Exception) -> Dict[str, Union[str, DAG]]:
```

Comment:
why `Dict[str, Union[str, DAG]]` and not just `Dict[str, DAG]`?

Reply:
I re-used the return signature of the `build` method that's below that one, as they return basically the same

`def build(self) -> Dict[str, Union[str, DAG]]:`

Reply:
well, in that one I think it is not accurate as well, dag is always instantiated as a `DAG` instance, not `string.